### PR TITLE
Add typings for launchpad

### PR DESCRIPTION
**Pull request by rictic**
case 1. Add a new type definition.
- [x] checked compilation succeeds with `--target es6` and `--noImplicitAny` options.
- [x] has correct [naming convention](http://definitelytyped.org/guides/contributing.html#naming-the-file)
- [x] has a [test file](http://definitelytyped.org/guides/contributing.html#tests) with the suffix of  `-tests.ts` or `-tests.tsx`.

One definite known issue, and one potential issue with this PR that I'd like advice with:

This PR and its test compile correctly when I just use `tsc` on the command line, but in travis or with `npm test` the test file is unable to find the module `"launchpad"`.

I'm also sending this PR to the types-2.0 branch, as I'd like to be able to install it from npm `@types/launchpad`.

### CODE REVIEW

File: launchpad/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "target": "es6",
+        "noImplicitAny": true,
+        "strictNullChecks": true,
```
Comment (mhegazy) [inline launchpad/tsconfig.json]: can you add `types: []` as well.

File: launchpad/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,108 @@
+// Type definitions for launchpad
+// Project: https://www.npmjs.com/package/launchpad
+// Definitions by: Peter Burns <https://github.com/rictic>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare module "launchpad" {
```
Comment (mhegazy) [inline launchpad/index.d.ts]: do not use external ambient module declarations. just write it as a normal ES6 module, with top level exports. this avoids polluting the global namespace and conflicts down the road if two versions of the module were loaded.

File: launchpad/tsconfig.json (hunk 1 above)
Comment (rictic) [inline launchpad/tsconfig.json]: Sure thing, done.

File: launchpad/index.d.ts (hunk 2 above)
Comment (rictic) [inline launchpad/index.d.ts]: Done

**Comment by rictic** [comment]
Comments addressed. `npm test` still fails as the test can't find the module `'launchpad'`.

**Comment by mhegazy** [comment]
can you refresh the PR one more time. the CI server should be working now, but the branch needs to be touched for it to run again.

**Comment by rictic** [comment]
Rebased against 2.0 branch, PTAL

**Pull request merged**